suite-native/app/package.json

```diff
@@ -114,7 +114,7 @@
         "react-native": "0.76.1",
         "react-native-edge-to-edge": "^1.3.1",
         "react-native-gesture-handler": "^2.21.0",
-        "react-native-keyboard-aware-scroll-view": "0.9.5",
+        "react-native-keyboard-controller": "patch:react-native-keyboard-controller@npm%3A1.16.5#~/.yarn/patches/react-native-keyboard-controller-npm-1.16.5-356a673ae8.patch",
```

Comment:
This doesn't look right, other patched packages have only the version defined in package.json